### ISSUE TITLE
Why quilt?

### ISSUE THREAD

**Issue by lynrayy** [LambdAurora/LambdaBetterGrass#96]
Which features quilt have (and fabric not) that you decided to reject big part of your users on fabric?

**Comment by PieyIsAPie** [comment]
quilt can load all fabric mods

> Which features quilt have (and fabric not) that you decided to reject big part of your users on fabric?

Quilt has the resource loading APIs the mod needs.

**Comment by lynrayy** [comment]
Fabric have all of those

**Comment by intergrav** [comment]
Fabric does not, what are you talking about
Quilt seems to have made this mod significantly easier to port to later versions. If you want to use the mod, just use Quilt! It supports Fabric mods anyway.

**Comment by adaliszk** [comment]
While quilt does load most fabric mods, it does not run them equally well; some do nothing, and others just corrupt your world. Once Quilt is out from beta, I suppose it will be a seamless experience, but for now, it's very early to burn bridges if there is a way to support Fabric.

However, it is your call as the Developer; just please be patient with us, who want to understand your standpoint!

**Comment by Oliver-makes-code** [comment]
> some do nothing, and others just corrupt your world. 

I've not had a single mod not work or corrupt a world outside of the fault of the developer of said mod. I helped organize an event with over 500 mods (BlanketCon), with most of them being fabric mods

> Once Quilt is out from beta, I suppose it will be a seamless experience

It already is pretty much a seamless experience, just replace fapi with qfapi and install quilt instead of fabric

**Comment by sylv256** [comment]
can we just start deleting issues like this? it's clogging up issue-space and is really annoying

**Comment by adaliszk** [comment]
These things are better as discussions anyway 😉

**Comment by Zerogoki00** [comment]
Yes, quilt is only in beta and it can corrupt your world. Fabric is very stable
That's why i'm not risking it and I stop using this mod. Will wait for Fabric alternative

**Comment by MimzyFang** [comment]
> Yes, quilt is only in beta and it can corrupt your world. Fabric is very stable
> That's why i'm not risking it and I stop using this mod. Will wait for Fabric alternative 

I've been using quilt for a while now.
From my experience it will not corrupt anything

And while I had some issues with the quilted fabric api's once I reported the issues and they got fixed 

So if you have any mod that crashes the game on quilt but works fine on fabric go report it as an issue 

Quilt is really stable and no one should be worried when using it 

If you are worried about corruption. have backups that's something you should have regardless of what modloader you use anyway

**Comment by intergrav** [comment]
> Yes, quilt is only in beta and it can corrupt your world.

do you have any evidence that Quilt has done it before? And yes I definitely agree with @Fauli1221, you should be keeping backups in any case with modding, even Fabric or Forge

**Comment by Magicrafter13** [comment]
> quilt can load all fabric mods

Yeah except when it can't. Oh yeah and when the Quilt-Fabric-API is stuck on 1.20.1 while everyone else is moving on from 1.20.2 to 1.20.3/1.20.4...

I haven't had any issues actually *using* quilt, but lets not act like using it is a perfect drop-in replacement with no reason to use fabric ever again. Because it's not.

**Comment by xbxct** [comment]
If there are incompatibilities, it is definitely on the part of the developer who limited the mod's availability in *version.

**Comment by Magicrafter13** [comment]
> If there are incompatibilities, it is definitely on the part of the developer who limited the mod's availability in *version.

Incorrect. In a perfect world yes, but Mojang/Microsoft do not follow proper Semantic Versioning. Mods can and do break compatibility between minor version number bumps. I'm well aware of modifying the version compatibility string to include a desired version as a workaround. It does not always work.

**Comment by lynrayy** [comment]
Also, quilt dead

**Comment by xbxct** [comment]
> Кроме того, лоскутное одеяло мертво

why?

**Comment by lynrayy** [comment]
> > Кроме того, лоскутное одеяло мертво
> 
> why?

Nobody really using it. Also, why your quote is in russian?

**Comment by xbxct** [comment]
> > > Кроме того, лоскутное одеяло мертво
> > 
> > 
> > why?
> 
> Nobody really using it. Also, why your quote is in russian?
Without a translator, my words are displayed in English, I can't say anything bad about qilt.

**Comment by xbxct** [comment]
Looks like this is a bug with data spoofing when using the translator

**Comment by LambdAurora** [comment]
Quilt being effectively dead, there is no longer any reason to keep this mod a Quilt exclusive.

Future updates will target Fabric. Current targets are 1.20.1, 1.21.1, 1.21.5, and 1.21.8+.

**Comment by lynrayy** [comment]
LMAO
Silly libertarian finally found out that here is no reason to use stillborn project.

**Comment by lynrayy** [comment]
Thank you for updates bro. Really.
